lib/commands/import.js

```diff
@@ -61,6 +63,18 @@ const serviceMapping = {
     const processFn = service.processStream.bind(service)
     const finishFn = () => log.info(service.summaryReport(options.from))
     return [ options, 'categories.*', processFn, finishFn ]
+  },
+  [allowedTypes.discount]: options => {
+    const service = new ProductDiscountImport(log,
+        Object.assign({}, options.config, {
```

Comment:
Yeah, but just because we use the same conventions in all the libraries. Maybe we can just require a specific API and we can simplify the setup here...